CHANGELOG

```diff
@@ -1,12 +1,19 @@
+1.2.0 (2020-04-24)
+---------------------------------------
```

Comment:
The hyphens are to be aligned with the release name, as stated [here](https://github.com/snowplow/beam-enrich/pull/31#discussion_r414777746)